plugins/argentina-afip-invoices/src/hook_install.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Service\TemplateRenderer;
+use Ubnt\UcrmPluginSdk\Exception\ConfigurationException;
+use Ubnt\UcrmPluginSdk\Security\PermissionNames;
+use Ubnt\UcrmPluginSdk\Service\UcrmApi;
+use Ubnt\UcrmPluginSdk\Service\UcrmOptionsManager;
+use Ubnt\UcrmPluginSdk\Service\PluginConfigManager;
+use Ubnt\UcrmPluginSdk\Service\UcrmSecurity;
+
+chdir(__DIR__);
+
+require_once __DIR__ . '/vendor/autoload.php';
+require __DIR__ . '/vendor/autoload.php';
+include __DIR__ . '/classes/Argentinacae/funciones.php';
+
+define('DEBUG', false); //Change to true debug comments get shown
+
+if (DEBUG) echo 'estoy en instalar/actualizar custom attributes';
+// directorio actual
+if (DEBUG) echo getcwd() . "\n";
+// Retrieve API connection.
+$api = UcrmApi::create();
+
+// Ensure that user is logged in and has permission to view invoices.
```

Comment:
Please note, that this check will never pass, because hooks are executed as a background process without any user. Same goes for the update hook.

Reply:
@keksa You're right i didn't realize that. Files updated and uploaded!

Thanks for your comment